src/Controls/samples/Controls.Sample/Pages/Controls/PickerPage.xaml.cs

```diff
@@ -112,6 +115,54 @@ void UpdatePickerBindingContext()
 				}
 			};
 		}
+
+		void OnUpdateHorizontalOptionsClicked(object sender, EventArgs e)
+		{
+			switch (_horizontalOptionsIndex)
+			{
+				case 0:
+					AlignmentPicker.HorizontalOptions = LayoutOptions.Start;
+					break;
+				case 1:
+					AlignmentPicker.HorizontalOptions = LayoutOptions.Center;
+					break;
+				case 2:
+					AlignmentPicker.HorizontalOptions = LayoutOptions.End;
+					break;
+				case 3:
+					AlignmentPicker.HorizontalOptions = LayoutOptions.Fill;
+					break;
+			}
+
+			_horizontalOptionsIndex++;
+
+			if (_horizontalOptionsIndex == 4)
+				_horizontalOptionsIndex = 0;
+		}
+
+		void OnUpdateVerticalOptionsClicked(object sender, EventArgs e)
+		{
+			_verticalOptionsIndex++;
+
+			switch (_verticalOptionsIndex)
+			{
+				case 0:
+					AlignmentPicker.VerticalOptions = LayoutOptions.Start;
+					break;
+				case 1:
+					AlignmentPicker.VerticalOptions = LayoutOptions.Center;
+					break;
+				case 2:
+					AlignmentPicker.VerticalOptions = LayoutOptions.End;
+					break;
+				case 3:
+					AlignmentPicker.VerticalOptions = LayoutOptions.Fill;
+					break;
+			}
+
+			if (_verticalOptionsIndex == 4)
+				_verticalOptionsIndex = 0;
+		}
```

Comment:
Maybe move the switch to a share method since they are the same?
```suggestion
       static LayoutOptions LayoutFromIndex (int index) => index switch
       {
              0 => LayoutOptions.Start,
              1 => LayoutOptions.Center,
              2 => LayoutOptions.End,
              3 => LayoutOptions.Fill,
              _ => throw new ArgumentOutOfRangeException(nameof(index)),
       }
		void OnUpdateHorizontalOptionsClicked(object sender, EventArgs e)
		{
			AlignmentPicker.HorizontalOptions = LayoutFromIndex (_horizontalOptionsIndex);
			_horizontalOptionsIndex++;

			if (_horizontalOptionsIndex == 4)
				_horizontalOptionsIndex = 0;
		}

		void OnUpdateVerticalOptionsClicked(object sender, EventArgs e)
		{
			_verticalOptionsIndex++;
			AlignmentPicker.VerticalOptions =  LayoutFromIndex (_verticalOptionsIndex);
			
			if (_verticalOptionsIndex == 4)
				_verticalOptionsIndex = 0;
		}
```

Also I have noticed that the ++ on the _horizontal index is after the switch, and the vertical one before the switch, is this intended? Cause it looks to me that before the switch is not what we want, but I might be mistaken.

Reply:
Created a shared method and fixed a mistake.